proto/cosmos/autocli/v1/options.proto

```diff
@@ -101,9 +101,6 @@ message FlagOptions {
   // default_value is the default value as text.
   string default_value = 4;
 
-  // default value is the default value as text if the flag is used without any value.
-  string no_opt_default_value = 5;
-
   // deprecated is the usage text to show if this flag is deprecated.
   string deprecated = 6;
```

Comment:
Actually, as we are deleting a field, maybe we should rename their identifier too, given it has not been yet released. cc @JeancarloBarrios

Reply:
True I missed that thanks !!

Reply:
I would suggest we don't do this actually... so the issue is that this is part of the `api` module not 0.47 at all. So backporting doesn't really do anything. The thing that we need to do is tag `api` and then backport the required API module version to 0.47. Removing the field is one thing, but changing the numbers really would break any case that someone built their binary with an earlier version of `api`. This is maybe the first case in the wild where the issues raised in ADR 046 can be observed. Let's discuss a tagging/versioning strategy next week.

Reply:
This makes sense!
One small unrelated issue is that now api supports CometBFT and v0.47 is still using Tendermint. So tagging a new version from main may be tricky.